src/components/LandingPageComponents/Stats.js

```diff
@@ -1,490 +1,464 @@
-import React, {useState, useEffect} from 'react'
+import React, { useState, useEffect } from 'react'
 import styled from 'styled-components'
+import { gql } from 'graphql-request'
+import { ChainId } from '@swapr/sdk'
+
 import { gradients, breakpoints } from '../../utils/theme'
 import { StatsContent } from '../../utils/ui-constants'
 import { toClassName } from './../../utils/helper-functions'
 import Layout from './layout/Layout'
-import TextyAnim from 'rc-texty';
-
-import { ApolloClient, InMemoryCache, gql } from '@apollo/client';
+import TextyAnim from 'rc-texty'
+import { immediateSubgraphClients } from '../../apollo/client'
 
-const apiUrlList = [
-    'https://api.thegraph.com/subgraphs/name/dxgraphs/swapr-arbitrum-one-v3',
-    'https://api.thegraph.com/subgraphs/name/dxgraphs/swapr-xdai-v2',
-    'https://api.thegraph.com/subgraphs/name/dxgraphs/swapr-mainnet-v2'
+const subgraphApiClients = [
+  immediateSubgraphClients[ChainId.ARBITRUM_ONE],
+  immediateSubgraphClients[ChainId.XDAI],
+  immediateSubgraphClients[ChainId.MAINNET]
 ]
 
-const tokensQuery = `{
-    swaprFactories (first:1000) {
-        txCount
-        totalVolumeUSD
+const tokensQuery = gql`
+  {
+    swaprFactories(first: 1000) {
+      txCount
+      totalVolumeUSD
     }
-}`;
-
+  }
+`
 
-const retrieveData = async (url) => {
-
-    const client = new ApolloClient({
-        uri: url, 
-        cache: new InMemoryCache(),
-    });
-
-    let asyncClient = await client
-        .query({
-            query: gql(tokensQuery)
-        })
-        .then((data) => {
-            // console.log('Subgraph data: ', data)
-            return data;
-        })
-        .catch((err) => {
-            console.log('Error fetching data: ', err);
-            return err;
-        });
-    return asyncClient;
+const retrieveData = async client => {
+  return await client
+    .request(tokensQuery)
+    .then(data => {
+      return data
+    })
+    .catch(err => {
+      console.log('Error fetching data: ', err)
+    })
 }
 
-
 const Stats = () => {
+  const [failedToUpdate, setFailedToUpdate] = useState(false)
+  const [tvl, setTvl] = useState(0)
+  const [swaprPrice, setSwaprPrice] = useState(0)
+  let [tx, setTx] = useState(0)
+  let [totalVolumeUSD, setTotalVolumeUSD] = useState(0)
+  const [isChartActive, setIsChartActive] = useState(false)
 
-    let [tvl, setTvl] = useState(0);
-    let [swaprPrice, setSwaprPrice] = useState(0);
-    let [tx, setTx] = useState(0);
-    let [totalVolumeUSD, setTotalVolumeUSD] = useState(0);
-    let [isChartActive, setIsChartActive] = useState(false);
+  useEffect(() => {
+    const tvlPromise = fetch('https://api.llama.fi/tvl/swapr')
+    tvlPromise
+      .then(data => {
+        return data.json()
+      })
+      .then(decodedTvl => {
+        setTvl((decodedTvl / 1000000).toFixed(1))
+      })
+      .catch(err => {
+        console.log('Error fetching data: ', err)
```

Comment:
I think this is clearer

```suggestion
        console.error('Error fetching Swapr TVL: ', err)
```

Reply:
I'll change it. Thanks.